.circleci/config.yml

```diff
@@ -87,6 +90,11 @@ jobs:
     <<: *container_config_node
     steps:
       - *attach_workspace
+      - browser-tools/install-chrome
+      - browser-tools/install-chromedriver
+      - run: 
+          name: Install puppeteer chromium missing dependencies
+          command: apt-get install -y libxss1 
```

Comment:
**suggestion:** it looks like `libxss1` is already installed in the latest `12.22-browsers` image: https://github.com/CircleCI-Public/cimg-node/blob/master/12.22/browsers/Dockerfile#L50

Reply:
thank you, upgrading 12.18-browsers -> 12.22-browsers fixed the problem I was facing

Reply:
🤔  I'm still baffled as to why 12.18 threw `  /home/circleci/project/build/node_modules/puppeteer/.local-chromium/linux-756035/chrome-linux/chrome: error while loading shared libraries: libXss.so.1: cannot open shared object file: No such file or directory
`
when it has libxss1 installed (here's [the dockerfile diffs](https://www.diffchecker.com/bScykl6w))

Reply:
Where did you get the 12.18 dockerfile from? Couldn't see it on GitHub but running `docker inspect` it doesn't look like it installs `libxss1` like the 12.22 dockerfile does

Reply:
I had to fork down the repo to [generate the dockerfile](filters) for 12.22, but interesting that you can't see it has libxss1 in 12.18 through docker inspect

Reply:
The link isn't working but they seem to have changed the scripts they use to generate their docker images since they pushed the 12.18 image so that might explain the discrepancy

Reply:
I just tested it with docker and can confirm that I see libxss1 in cimg/node:12.22-browsers but not in cimg/node:12.18-browsers